src/fortran/epos/sources/epos-random.f

```diff
@@ -191,66 +191,6 @@ subroutine ranfini(seed,iseq,iqq)
       return
       end
 
-C=======================================================================
-      SUBROUTINE RMMARD( RVEC,LENV,ISEQ )
-C-----------------------------------------------------------------------
-C  C(ONE)X
-C  R(ANDO)M (NUMBER GENERATOR OF) MAR(SAGLIA TYPE) D(OUBLE PRECISION)
-C
-C  THESE ROUTINES (RMMARD,RMMAQD) ARE MODIFIED VERSIONS OF ROUTINES
-C  FROM THE CERN LIBRARIES. DESCRIPTION OF ALGORITHM SEE:
-C               http://consult.cern.ch/shortwrups/v113/top.html
-C  IT HAS BEEN CHECKED THAT RESULTS ARE BIT-IDENTICAL WITH CERN
-C  DOUBLE PRECISION RANDOM NUMBER GENERATOR RMM48, DESCRIBED IN
-C               http://consult.cern.ch/shortwrups/v116/top.html
-C  ARGUMENTS:
-C   RVEC   = DOUBLE PREC. VECTOR FIELD TO BE FILLED WITH RANDOM NUMBERS
-C   LENV   = LENGTH OF VECTOR (# OF RANDNUMBERS TO BE GENERATED)
-C   ISEQ   = # OF RANDOM SEQUENCE
-C
-C  VERSION OF D. HECK FOR DOUBLE PRECISION RANDOM NUMBERS.
-C  ADAPTATION  : T. PIEROG    IK  FZK KARLSRUHE FROM D. HECK VERSION
-C  DATE     : Feb  17, 2009
-C-----------------------------------------------------------------------
-      IMPLICIT NONE
-      INTEGER          KSEQ
-      PARAMETER        (KSEQ = 2)
-      COMMON /CRRANMA3/CD,CINT,CM,TWOM24,TWOM48,MODCNS
-      DOUBLE PRECISION CD,CINT,CM,TWOM24,TWOM48
-      INTEGER          MODCNS
-      COMMON /CRRANMA4/C,U,IJKL,I97,J97,NTOT,NTOT2,JSEQ
```

Comment:
Please don't remove this. The idea is to keep "epos-random.f" untouched, because this is not our code.